code/network/multiutil.cpp

```diff
@@ -1724,7 +1725,7 @@ void multi_free_active_games()
 			backup = moveup;
 			moveup = moveup->next;
 			
-			vm_free(backup);
+			delete backup;
```

Comment:
This function looks like a candidate for smart pointers, but that's out of scope for this PR.